scripts/result_scanner/result_scanner.rb

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env ruby
+require 'json'
+
+module ResultScanner
+
+  Result = Struct.new(:name, :confidence, :risk, :url, :param, :evidence, :alert)
+
+  class << self
+
+    # Call Me.
+    def summarize_results(last_run_path, current_run_path, output_path)
+      @final_json_out = {}
+      @final_text_out = "Completed scan of #{json_files_from_dir(current_run_path).size} properties:"
```

Comment:
Nope.  I think I `@`'d them just for syntax highlighting :stuck_out_tongue:
I'll fix.